src/FSharp.Core/prim-types.fs

```diff
@@ -221,7 +221,7 @@ namespace Microsoft.FSharp.Core
         member _.Minor = minor
         member _.Release = release
 
-    [<AttributeUsage(AttributeTargets.All, AllowMultiple=false)>]
+    [<AttributeUsage(AttributeTargets.All, AllowMultiple=true)>]
```

Comment:
Compiler actually checks that `CompilationMappingAttribute` is not usable from user-code, it is reserved for compiler use only.